clickhouse_cli/cli.py

```diff
@@ -154,6 +154,7 @@ def load_config(self):
         # forcefully disable `highlight_output` in (u)rxvt (https://github.com/hatarist/clickhouse-cli/issues/20)
         self.highlight_output = False if os.environ.get('TERM', '').startswith('rxvt') else self.config.getboolean('main', 'highlight_output')
         self.highlight_truecolor = self.config.getboolean('main', 'highlight_truecolor') and os.environ.get('COLORTERM')
+        self.highlight_theme = self.config.get('main', 'highlight_theme')
```

Comment:
should've also use fallback value and/or add the option to the sample configuration file